ProcessLib/LiquidFlow/LiquidFlowMaterialProperties.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   LiquidFlowMaterialProperties.h
+ *
+ * Created on August 18, 2016, 11:03 AM
+ */
+
+#ifndef OGS_LIQUIDFLOWMATERIALPROPERTIES_H
+#define OGS_LIQUIDFLOWMATERIALPROPERTIES_H
+
+#include <memory>
+
+#include "MaterialLib/Fluid/FluidPropertyHeaders.h"
+#include "MaterialLib/PorousMedium/PorousPropertyHeaders.h"
+
+#include "MeshLib/PropertyVector.h"
+
+namespace MeshLib
+{
+class Mesh;
+}
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+class Porosity;
+class Storage;
+}
+}
+
+namespace ProcessLib
+{
+namespace LiquidFlow
+{
+struct LiquidFlowMaterialProperties
+{
+    typedef MaterialLib::Fluid::FluidProperty::ArrayType ArrayType;
+
+    explicit LiquidFlowMaterialProperties(BaseLib::ConfigTree const& config);
+
+    /**
+     * \brief Compute the coefficient of the mass term by
+     *      \f[
+     *           n \frac{partial \rho_l}{\partial p} + \beta_s
+     *      \f]
+     *     where \f$n\f$ is the porosity, \f$rho_l\f$ is the liquid density,
+     *     \f$bata_s\f$ is the storage.
+     * \param var4porosity       The first variable for porosity model, and it could be
+     *                           saturation, and invariant of stress or strain.
```

Comment:
The name is replaced with a meaningful one. This augment just passes a double type value, and hereby  it is unnecessary to distinguish what is the value for. With the value by this argument, the function just calls the base of polymorphic classes  to compute a porosity.

The comment is improved according the above explanation.